src/Services/Air/Air.js

```diff
@@ -248,8 +246,7 @@ module.exports = (settings) => {
                   })
                   .then(parsers.bookingPnr)
                   .then(pnr => localTerminal.closeSession()
-                    .then(() => ({ ...line, pnr }))
-                  );
+                    .then(() => ({ ...line, pnr })));
```

Comment:
Why do you prefer to close all the braces at once?
It's then hard to see what started where